linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/bash/ubuntu2004.sh

```diff
@@ -0,0 +1,5 @@
+# platform = multi_platform_ubuntu
```

Comment:
you can drop this, we are using the templated bash, unless you see something wrong with it

Reply:
Same as below, the templated remediation is not passing within DISA Benchmark